### ISSUE TITLE
Since version 12.4 the first example in the readme gives an error: KeyError: 'emb_dropout'

### ISSUE THREAD

**Issue by wingedsheep** [lucidrains/x-transformers#54]
![image](https://user-images.githubusercontent.com/2526796/126061383-9789160a-be67-4edc-90db-1ce20344960d.png)

Since version 12.4 the first example in the readme gives an error: KeyError: 'emb_dropout'

```
import torch
from x_transformers import XTransformer

model = XTransformer(
    dim = 512,
    enc_num_tokens = 256,
    enc_depth = 6,
    enc_heads = 8,
    enc_max_seq_len = 1024,
    dec_num_tokens = 256,
    dec_depth = 6,
    dec_heads = 8,
    dec_max_seq_len = 1024,
    tie_token_emb = True      # tie embeddings of encoder and decoder
)

src = torch.randint(0, 256, (1, 1024))
src_mask = torch.ones_like(src).bool()
tgt = torch.randint(0, 256, (1, 1024))
tgt_mask = torch.ones_like(tgt).bool()

loss = model(src, tgt, src_mask = src_mask, tgt_mask = tgt_mask) # (1, 1024, 512)
loss.backward()
```

**Comment by lucidrains** [comment]
@wingedsheep fixed in 0.14.1!

**Comment by wingedsheep** [comment]
Tested in 0.14.1
